frontend/src/Pages/EventAttendance.tsx

```diff
@@ -208,8 +222,9 @@ export default function EventAttendance() {
                 </div>
                 <button
                     onClick={() => void handleMarkAllPresent()}
-                    disabled={anyRowSelected}
-                    className={`button ${anyRowSelected ? `bg-gray-400 cursor-not-allowed` : ``}`}
+                    disabled={anyRowSelected || !canEditAttendance}
+                    className={`button  tooltip tooltip-left ${anyRowSelected ? `bg-gray-400 cursor-not-allowed` : ``}`}
```

Comment:
@corypride We should be using one of the `bg-grey-{1,2,3,4} variants and not `bg-gray-400`. You can find these in `tailwind.config.cjs`.